src/pages/calendar/components/Section.tsx

```diff
@@ -77,21 +133,11 @@ export function SectionInfo({
             <Badge colorScheme="green" mx="1">
               {instructionalMethod}
             </Badge>
-            {isASYNC && (
-              <Badge colorScheme="cyan" mx="1">
-                Asynchronous
-              </Badge>
-            )}
-            {isSENG && (
-              <Badge colorScheme="orange" mx="1">
-                SENG ONLY
-              </Badge>
-            )}
-            {isCSC && (
-              <Badge colorScheme="yellow" mx="1">
-                CSC ONLY
-              </Badge>
-            )}
+            {badges.map((badges, index) => (
```

Comment:
I'd use name here instead of index as its preferred over using index and name is unique here.